app/uk/gov/hmrc/economiccrimelevyregistration/navigation/contacts/AddAnotherContactPageNavigator.scala

```diff
@@ -34,6 +34,15 @@ class AddAnotherContactPageNavigator extends PageNavigator {
       case _           => routes.JourneyRecoveryController.onPageLoad()
     }
 
-  override protected def navigateInCheckMode(registration: Registration): Call = ???
+  override protected def navigateInCheckMode(registration: Registration): Call =
+    registration.contacts.secondContact match {
+      case Some(true)  =>
+        registration.contacts.secondContactDetails.name match {
```

Comment:
Should probably match on all 4 of the second contact details here (name, role, email and number) - because you could specify the name, go back, then go forward and it would take you to check your answers